raocp/core/problem_spec.py

```diff
@@ -0,0 +1,205 @@
+import numpy as np
+import raocp.core.scenario_tree as core_tree
+
+
+class RAOCP:
+    """
+    Risk-averse optimal control problem creation
+    """
+
+    def __init__(self, last_nonleaf_node, last_leaf_node,
+                 state, ip, system_dynamics, input_dynamics,
+                 cost_type, Q, R, Pf,
+                 risk_type, alpha, E, F, Kone, b):
```

Comment:
Change `Kone` to `kone` (and why not `cone`?) Follow the PEP8 naming convention.

Reply:
Should't `Kone` be an instance of a certain class?

Reply:
we use calligraphic K in our notes to denote the cone, but I didn't want to use K so I used Kone - but yes, cone makes more sense

Reply:
have we decided how we will denote the cone ?